<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutForm</class>
 <widget class="QWidget" name="AboutForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLabel" name="lblVersion">
     <property name="text">
      <string>Version:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="txtAbout"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblClt">
     <property name="text">
      <string>Client:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="lblDevelpedBy">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lblDev">
     <property name="text">
      <string>Outside developers:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="7">
    <widget class="QGraphicsView" name="gphView"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lblCpn">
     <property name="text">
      <string>Company:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="lblCompany">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="lblVersionValue">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="lblClient">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
